ibm/mas_devops/roles/cp4d_wds/templates/cr.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: discovery.watson.ibm.com/v1
+kind: WatsonDiscovery
+metadata:
+  annotations:
+    oppy.ibm.com/disable-rollback: 'true'
+  name: wd
+  namespace: {{ cpdnamespace }}
+spec:
+  license:
+    accept: true
+  version: "{{ wdsversion is defined | ternary(wdsversion,'4.0.4') }}"
```

Comment:
If we move the default 4.0.4 to the default/main.yml then we don't need it here.

Reply:
move it to default.yml